### PR TITLE
[DX-198] Watch .github.io folders

### PR DESCRIPTION
**Pull request by matteofigus**
This affects the opencomponents.github.io repo

### CODE REVIEW

File: src/resources/settings.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ module.exports = {
   configFile: {
     src: './oc.json'
   },
+  filesToIgnoreOnDevWatch: /node_modules|package\.tar\.gz|_package|\.sw[op]|\.git\/|\.DS_Store|oc\.json/,
```
Comment (matteofigus) [inline src/resources/settings.js]: I fixed by adding a final `\/` after the `.git`

File: test/unit/resources-settings.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      it('should not ignore it', () => {
+        const result = execute('/path/to/component/server.js');
+        expect(result).to.be.false;
+      });
+    });
+
+    describe('when a file in a github.io repo changes', () => {
+      it('should not ignore it', () => {
+        const result = execute(
+          '/path/to/opencomponents.github.io/components/landing-page/server.js'
```
Comment (matteofigus) [inline test/unit/resources-settings.js]: This was before matching the `.git` and was failing

**Comment by Kimtaro** [comment]
@matteofigus Regex is A-OK 👍

**Comment by matteofigus** [comment]
Thanks for the review @Kimtaro

**Pull request merged**